src/learn/LearnDashboard.js

```diff
@@ -60,7 +60,7 @@ function LearnDashboard(props) {
           className="col-xs-12 col-sm-4"
           style={{ display: "flex", flexDirection: "column" }}
         >
-          <h2>{I18n.get("myMentors")}</h2>
+          {props.coaches.fName && <h2>{I18n.get("myMentors")}</h2>}
```

Comment:
Off the top of my head, I don't think this is correct - I'm not sure you actually pulled the code down on your local machine ;) If you look at the next line of code, where we run `coaches.map` you will see that coaches is an array. So, logically, it will not have a property called `fName`. Perhaps a better way would be for you to check `props.coaches.length > 0` or something to that extent. Regardless, will you please upload a screenshot with the next revision of the PR showing that the UI is in fact displaying correctly?

Reply:
Hello,

I see, thanks for the explanation.The code is definitely on my local machine lol. I'm new to Js so this is helping me get better. I've made some changes and will make the next revision of the PR.